ggircs-app/app/server/schema.graphql

```diff
@@ -948,6 +948,11 @@ type Query implements Node {
   session: JwtToken
 }
 
+"""
```

Comment:
not sure why this moved?

Reply:
This is odd, and it generated a huge change in the schema.json file...I wonder what could have caused that